GoogleSignIn/Sources/GIDDataFetcher/Implementations/GIDDataFetcher.m

```diff
@@ -0,0 +1,40 @@
+#import "GoogleSignIn/Sources/GIDDataFetcher/Implementations/GIDDataFetcher.h"
+
+#ifdef SWIFT_PACKAGE
+@import AppAuth;
+@import GTMAppAuth;
+#else
+#import <AppAuth/AppAuth.h>
+#import <GTMAppAuth/GTMAppAuth.h>
+#endif
+
+NS_ASSUME_NONNULL_BEGIN
+
+// Maximum retry interval in seconds for the fetcher.
+static const NSTimeInterval kFetcherMaxRetryInterval = 15.0;
+
+@implementation GIDDataFetcher
+
+- (void)startFetchURL:(NSURL *)URL
+    fromAuthState:(OIDAuthState *)authState
+      withComment:(NSString *)comment
+       completion:(void (^)(NSData *, NSError *))completion {
+  NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:URL];
+  GTMSessionFetcher *fetcher;
+  GTMAppAuthFetcherAuthorization *authorization =
+      [[GTMAppAuthFetcherAuthorization alloc] initWithAuthState:authState];
+  id<GTMSessionFetcherServiceProtocol> fetcherService = authorization.fetcherService;
```

Comment:
The fetcherService appears to be always nil if GTMAppAuthFetcherAuthorization is initialized this way. Can you please double check?